docker-compose.yml

```diff
@@ -34,14 +33,15 @@ services:
       # Networking:
       - dbms.connector.bolt.listen_address=0.0.0.0:7687
     healthcheck:
-        test: ["CMD", "curl", "-f", "http://localhost:7474"]
+        test: ["CMD", "wget", "--no-verbose", "http://localhost:7474"]
```

Comment:
wget works better than curl in reporting health for some reason. curl would get stuck on 'starting' status.